packages/components/input/src/InputElement.styles.ts

```diff
@@ -1,7 +1,7 @@
 import { cva, VariantProps } from 'class-variance-authority'
 
 export const inputElementStyles = cva(
-  ['text-neutral pointer-events-none absolute top-1/2 -translate-y-1/2'],
+  ['text-neutral pointer-events-none absolute top-[22px] -translate-y-1/2'],
```

Comment:
I need to improve this part. Fixed values shouldn't be used

Reply:
If it can't be eliminated, maybe at least we could use `calc()` to use the appropriate spacing token minus 2px to reach the same result. It would scale better the day we change token values :)

Reply:
Yup. Gonna ping @acd02 to check if he has a another idea

Reply:
I don't have a better idea for handling that use case 😄, so lgtm